### ISSUE TITLE
Need to keep the HTTP-connection alive in order to keep the authentication session alive

### ISSUE THREAD

**Issue by Borewit** [Borewit/musicbrainz-api#1]
####Description:
It looks like posting though the user/bot form API fails because the HTTP connection is not kept alive. 
The prescribed way to keep it alive for Axios, the underlying HTTP-client does not seem to work: axios/axios#1981.

#### Workaround:

Use Fiddler as proxy in-between:

``` JavaScript
const Fiddler = {
  host: '127.0.0.1',
  port: 8888
};

const config = {
  proxy: Fiddler,
  //...
}
```

Fiddler will keep the connection to MusicBrainz alive.

**Comment by Borewit** [comment]
Fixed in [v0.2.0](https://github.com/Borewit/musicbrainz-api/releases/tag/v0.2.0)

**Comment by Borewit** [comment]
Made mistake in the comment. I fxed it using [request](https://github.com/request/request)
Also tried Needle as well, but is had the same issue as Axio: tomas/needle#271.
